pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -130,14 +130,44 @@ class SELinuxEventDetails extends React.Component {
                     </div>
                 );
             }
+
+            // Formatted solution
+            let doElement = "";
+
+            // One line usually means one command
+            if (itm.doText && itm.doText.indexOf("\n") < 0)
+                doElement = <pre>{itm.doText}</pre>;
+
+            // There can be text with commands. Command always starts on a new line with '#'
+            // Group subsequent commands into one `<pre>` element.
+            if (itm.doText.indexOf("\n") >= 0) {
+                const parts = [];
+                const lines = itm.doText.split("\n");
+                let lastCommand = false;
+                lines.forEach(l => {
+                    if (l[0] == "#") { // command
+                        if (lastCommand)
+                            parts[parts.length - 1] += ("\n" + l.substr(2));
+                        else
+                            parts.push(l);
+                        lastCommand = true;
+                    } else {
+                        parts.push(l);
+                        lastCommand = false;
+                    }
+                });
+                doElement = <>{parts.map(p => p[0] == "#" ? <pre>{p.substr(2)}</pre> : <span>{p}</span>)}</>;
```

Comment:
This doesn't need a fragment wrapper, you can directly render arrays. However, the elements do need a key.

Can it ever happen that `p[0] == "#"`? These are cut off above, no?

Reply:
right, I already have keys locally, just did not get to pushing it.

> Can it ever happen that p[0] == "#"? These are cut off above, no?

No, the first command still has the '#', only the subsequent ones are removed. I need to keep track of which are command and which are just text, and thought that this would be the least hacky solution.

Reply:
Ah, thanks; please add a comment that explains this.